File: aman.c
#include <stdio.h>    
void accending();  
void decending();
int main()    
{   
  int arr[5];  
  int i; 

  printf("Enter elements of the array :\n");

  for(i=0;i<5;i++)
  {
    scanf("%d",&arr[i]);
  }
        
    int temp = 0;  
    char choice;
    
    int length = 10;    
           
    printf("Elements of original array: \n");    
    for (int i = 0; i < length; i++) {     
        printf("%d ", arr[i]);     
    }
    printf("\n");   
    printf("Enter your choice:");
    scanf("%c",&choice);
    if((choice == 'A') || (choice == 'a')){
        accending(arr);
    }
    else if ((choice == 'D')||(choice == 'd')){
        decending(arr);
    }
    
    return 0; 
    
}     

void accending(int a[]){
    // printf("accending");
    // printf("%d",&arr[5]);
    int temp=0;
    int length=5;
    //int length = sizeof(arr)/sizeof(arr[0]);  
    for (int i = 0; i < length; i++) {     
        for (int j = i+1; j < length; j++) {     
          if(a[i] > a[j]) {    
              temp = a[i];    
              a[i] = a[j];    
              a[j] = temp;    
          }     
        }     
    } 
    printf("Elements of array sorted in ascending order: \n");    
    for (int i = 0; i < length; i++) {     
        printf("%d ", a[i]);    
    }     
}


void decending(int a[]){
    // printf("accending");
    // printf("%d",&arr[5]);
    int temp=0;
    int length=5;
    //int length = sizeof(arr)/sizeof(arr[0]);  
    for (int i = 0; i < length; i++) {     
        for (int j = i+1; j < length; j++) {     
          if(a[i] < a[j]) {    
              temp = a[i];    
              a[i] = a[j];    
              a[j] = temp;    
          }     
        }     
    } 
    printf("Elements of array sorted in decending order: \n");    
    for (int i = 0; i < length; i++) {     
        printf("%d ", a[i]);    
    }     
}

File: aman.py
def accending_str(arr):
    temp=0
    for i in range(len(arr)):
        for j in range(len(arr)):
            if(int(arr[i]) < int(arr[j])):  
                temp = arr[i]    
                arr[i] = arr[j]    
                arr[j] = temp  
                
    # for i in range(len(arr)):
    #     print(arr,end=' ')
    print('Given array in accending order:')
    print(arr)


def decending_str(arr):   
    temp=0
    for i in range(len(arr)):
        for j in range(len(arr)):
            if(int(arr[i]) > int(arr[j])):  
                temp = arr[i]    
                arr[i] = arr[j]    
                arr[j] = temp  
                
    # for i in range(len(arr)):
    #     print(arr,end=' ')
    print('Given array in decending order:')
    print(arr)

# arr_len=int(input('enter length:'))
# for i in range(arr_len):
#     str=int(input('Enter the string:'))
# try:
#     str=input('Enter the string:').split(' ')
#     for i in str:
#         print(i)
#         # if (i != int(i)):
#         #     pass   
#         #     # raise ValueError
#         # else:
#         print('original array:')
#         # print(str)
#         sel=input("Select 'A' for accending ,Select 'D' for decending:")
#         if(sel=='A'or sel=='a'):
#             print(accending_str(str))
#         elif(sel=='D'or sel=='d'):
#             print(decending_str(str))
#         else:
#             print('wrong selection!')
# except ValueError:
#     print('Enter integer value only!')

str=input('Enter the string:').split(' ')
# for i in str:
#     print(i)

print('original array:')
print(str)
sel=input("Select 'A' for accending ,Select 'D' for decending:")
if(sel=='A'or sel=='a'):
    print(accending_str(str))
elif(sel=='D'or sel=='d'):
    print(decending_str(str))
else:
    print('wrong selection!')